src/main.rs

```diff
@@ -13,6 +16,29 @@ use std::collections::HashMap;
 
 use std::fs;
 
+struct TerminalOutput(bool);
+
+#[rocket::async_trait]
+impl<'r> FromRequest<'r> for TerminalOutput {
+    type Error = ();
+
+    async fn from_request(request: &'r Request<'_>) -> Outcome<Self, Self::Error> {
+        if let Some(user_agent) = request.headers().get_one("User-Agent") {
+            if user_agent.contains("curl") {
+                return Outcome::Success(TerminalOutput(true));
+            }
+        }
+
+        if let Some(media_type) = request.headers().get_one("Accept") {
+            if media_type.contains("text/plain") {
```

Comment:
This makes the site unusable when using the lynx browser. Maybe make sure that the media_type does not contain the "text/html" string too?

Reply:
How does a request from the lynx browser look? What breaks?

Reply:
this is the media_type: `text/html, text/plain, text/sgml, text/css, application/xhtml+xml, */*;q=0.01
`, which triggers the `.contains("text/plain")

Reply:
Why? 🤔 I'll add the `!media_type.contains("text/html")` check and hope it'll work better.

Reply:
yeah http headers are never consistent. Add that check and everything should work